frontend/src/components/ModalContent.vue

```diff
@@ -1,119 +1,116 @@
 <template>
   <v-card>
-    <v-card-title class="title-header">
-      <v-row>
-        <p class="modal-title">{{ selectedProject.name }}</p>
-        <v-card-actions>
-          <v-btn
-            class="mr-2"
-            absolute
-            fab
-            rounded
-            icon
-            large
-            right
-            @click="onClick()"
-          >
-            <v-icon size="3.5rem" color="white">close</v-icon>
-          </v-btn>
-        </v-card-actions>
-      </v-row>
+    <v-card-title class="container-title unified-padding">
+      <h1>{{ selectedProject.name }}</h1>
```

Comment:
Given our discussion of h1 tags we should decice whether using h1 here is appropriate at all. Whatever the class should be I wouldn't argue moving at least a port of it to the global class. Will keep you posted on the adjustments.

Reply:
I removed h1 tag to reserve it for singular use only - although multiple h1 tags don't matter anymore in html 5, because web crawlers use sections to understand the structure in more modular way (e.g. [Google](https://www.youtube.com/watch?v=WsgrSxCmMbM)) people still believe that it is the best practice. Basically the argument is that with the singular use you don't lose anything but you ay gain sth in particular cases. 

So I used h2 and h3 tags instead - both went to global classes.